controllers/vgmanager_test.go

```diff
@@ -0,0 +1,85 @@
+package controllers
+
+import (
+	"context"
+	"testing"
+
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	"gotest.tools/v3/assert"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	testNamespace = "default"
+)
+
+func newFakeLVMClusterReconciler(t *testing.T, objs ...client.Object) *LVMClusterReconciler {
```

Comment:
only a question, how different is this when compared to envtest, not the actual tests but the functionality this provides over envtest.

Reply:
It does not provide any advantage over envTest.   EnvTest should be the better way to right the tests compared to fake clients.